lib/dev/ui/prompt.rb

```diff
@@ -88,11 +88,25 @@ def ask(question, options: nil, default: nil, is_file: nil, allow_empty: true)
         #
         def confirm(question)
           puts_question("#{question} {{yellow:(choose with ↑ ↓ ⏎)}}")
-          InteractivePrompt.call(%w(yes no)) == 'yes'
+          interactively_ask(question, %w(yes no)) == 'yes'
         end
 
         private
 
+        def interactively_ask(question, options)
+          resp = InteractivePrompt.call(options)
+
+          # After interactive prompt, the questions are erased
+          # Return to the question line. The space is for initial padding we miss
+          print(ANSI.previous_line + ANSI.end_of_line + ' ')
```

Comment:
This feels like it should be part of `InteractivePrompt.call`.  Without this change, `InteractivePrompt.call` would leave a record of the choice made via the `>` and colour.  With this change as is, that record is lost.

Reply:
yea, I know. Question asking is a part of `prompt` though

Reply:
Hmm.  Maybe `InteractivePrompt` should be a private class nested under `Prompt` then.